include/swift/Runtime/Config.h

```diff
@@ -216,6 +216,7 @@
 // But some of the runtime entries are invoked directly from
 // the foundation. Therefore they should be visible.
 #define SWIFT_RT_ENTRY_VISIBILITY SWIFT_RUNTIME_EXPORT
+#define SWIFT_EXTERN_RT_ENTRY_VISIBILITY SWIFT_EXTERN_RUNTIME_EXPORT
```

Comment:
Those should remain simply `extern "C"`. Those exist simply to make the symbol name match the @_silgen_name used on the Swift side. They aren't exported anywhere so `__declspec` doesn't apply.